modules/meazyBidAdapter.js

```diff
@@ -0,0 +1,141 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
+
+const BIDDER_CODE = 'meazy';
+const PREBID_ENDPOINT = 'rtb-filter.meazy.co';
+const SYNC_ENDPOINT = '//sync.meazy.co/sync/iframe';
+const ENDPOINT_CONFIG = {
+  defaultCurrency: ['USD'],
+  availableSize: ['300x250', '320x480', '160x600']
+};
+
+const buildURI = (pid) => {
+  return `//${PREBID_ENDPOINT}/pbjs?host=${utils.getOrigin()}&api_key=${pid}`;
+}
+
+const validateSize = (size) => {
+  return ENDPOINT_CONFIG.availableSize.indexOf(size.join('x')) !== -1;
+}
+
+const buildImpression = (bidRequest) => {
+  const impression = {
+    id: utils.getUniqueIdentifierStr(),
+    tagid: bidRequest.adUnitCode,
+    banner: {
+      format: bidRequest.sizes.map(size => ({ w: size[0], h: size[1] }))
+    }
+  };
+
+  return impression;
+}
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {object} bid The bid to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    return !!bid.params.pid && bid.sizes.some(validateSize);
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {BidRequest[]} bidRequests A non-empty list of bid requests which should be sent to the Server.
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function(bidRequests, bidderRequest) {
+    const payload = {
+      id: bidRequests[0].bidId,
+      site: {
+        domain: utils.getOrigin(),
+        page: utils.getTopWindowUrl(),
+        ref: utils.getTopWindowReferrer()
```

Comment:
These two `getTopWindow` functions are slated to be deprecated in a future version of Prebid.  

There is an alternate property in the `bidderRequest` object that contains type of information and should be used instead of these functions.  The following link provides some additional details on the property:
http://prebid.org/dev-docs/bidder-adaptor.html#referrers

Can you please update these functions?